libqtile/bar.py

```diff
@@ -184,6 +184,7 @@ class Bar(Gap, configurable.Configurable, CommandObject):
         ("margin", 0, "Space around bar as int or list of ints [N E S W]."),
         ("border_color", "#000000", "Border colour as str or list of str [N E S W]"),
         ("border_width", 0, "Width of border as int of list of ints [N E S W]"),
+        ("floating", False, "Don't reserve screen space (bar will be drawn above windows)."),
```

Comment:
I think the words "floating" and "above" here are confusing because they already have specific uses in qtile. Could we use something like `"dont_reserve"` and "Don't reserve screen space for the bar in tiled layouts"?

Reply:
Ha. That's exactly what I want for with my first version of this and for the reasons you said. I went with floating because it felt neater but happy to change it.